tool/src/kotlin/mod.rs

```diff
@@ -66,7 +67,10 @@ impl KotlinConfig {
                 }
             }
             "use_finalizers_not_cleaners" => {
-                self.use_finalizers_not_cleaners = value.as_bool();
+                self.use_finalizers_not_cleaners = value.as_str().map(|val| val == "true");
```

Comment:
How we're parsing config values isn't correctly getting types. As far as I can see toml will only parse Tables and Arrays and not other values.